option optimize_for = LITE_RUNTIME;

message KingFightStageInfoRequest
{
 required uint32 id = 1;
 required uint32 refresh = 2;
}
